source/write-operations/insert-documents.txt

```diff
@@ -78,26 +71,31 @@ To learn more about document structure and rules, see the
 Insert One Document
 -------------------
 
-To add a single document to a MongoDB collection, call the ``insertOne()``
-method and pass the document you want to add. After you call the
-``insertOne()`` method, chain the ``subscribe()`` method and pass in a subscriber.
+To add a single document to a MongoDB collection, use the static
+``Mono.from`` method from the ``Mono`` ``Publisher`` to call the ``insertOne()``
```

Comment:
Q: can you desribe what is mean by `Mono` `Publisher`? Can the Monospace be combined to cover both words?

Reply:
It is a type of publisher implemented by the library called a mono publisher, which is in the next sentence. Do you think I need to elaborate more on details of the publisher?